pkg/apiserver/api_installer.go

```diff
@@ -78,11 +78,13 @@ func (a *APIInstaller) Install() (ws *restful.WebService, errors []error) {
 	}
 	sort.Strings(paths)
 	for _, path := range paths {
-		if err := a.registerResourceHandlers(path, a.group.Storage[path], ws, proxyHandler); err != nil {
+		apiResource, err := a.registerResourceHandlers(path, a.group.Storage[path], ws, proxyHandler)
+		if err != nil {
 			errors = append(errors, err)
 		}
+		apiResources = append(apiResources, *apiResource)
```

Comment:
I don't think apiResources needs to be a global, especially not if you're returning it from this function.